src/tools/genVersionHeader.pl

```diff
@@ -53,6 +56,7 @@
         my $hasmod = `darcs whatsnew --repodir="$opt_t" -l`;
         $opt_V .= '-dirty' unless $?;
     }
+    $cv = ""; # ToDo
```

Comment:
AI on RL to check and advise on darcs.

Reply:
Darcs code:
`darcs log --last 1 | grep Date: | sed -s 's/Date:   //'`

This takes the date of the last commit, while darcs may reorder commits, so commits further down may have newer dates. (But in principle this also applies to git, where commits could have been moved.)

Reply:
In case this isn't obvious, Ralph's `| grep ... | sed ...` part isn't portable to Windows, so would need to be converted into Perl.

Reply:
Committing the Perl version as:
```perl
   $cv = `darcs log --last 1`;
   $cv =~ s/ \A .* Date: \s+ (\N+) .* \z /$1/sx;
```